libminifi/src/core/controller/StandardControllerServiceNode.cpp

```diff
@@ -23,14 +23,11 @@
 namespace org::apache::nifi::minifi::core::controller {
 
 bool StandardControllerServiceNode::enable() {
+  controller_service_->setState(ENABLED);
   logger_->log_trace("Enabling CSN {}", getName());
-  if (active) {
-    logger_->log_debug("CSN {} is already enabled", getName());
-    return true;
-  }
-  Property property("Linked Services", "Referenced Controller Services");
-  if (getProperty(property.getName(), property)) {
-    for (const auto& linked_service : property.getValues()) {
+  if (auto linked_services = getAllPropertyValues("Linked Services")) {
+    active = true;
```

Comment:
What was the reason for this change? In PR https://github.com/apache/nifi-minifi-cpp/pull/1933 I have also have some changes for this state handling so we should align them.

Reply:
wow good catch, truth be told I have no idea... I think it might be some unfinished refactor when I wanted to remove the property sequence thingy in its entirety. reverted so it only includes the required changes in https://github.com/apache/nifi-minifi-cpp/commit/4716942ac598af876e0a093abcb963849546f5ec